scripts/rl_conclusion_registry.py

```
@@ -89,11 +89,34 @@ def load_registry(path: Path) -> JsonObject:
     return parsed
 
 
+def merge_normalized_record_maps(sources: Sequence[dict[str, JsonObject]]) -> dict[str, JsonObject]:
+    records: dict[str, JsonObject] = {}
+    for source in sources:
+        for conclusion_id, record in source.items():
+            if conclusion_id in records:
+                raise ConclusionRegistryError(f"duplicate conclusionId {conclusion_id!r} in conclusions payload")
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)